src/test/java/hudson/plugins/git/UserRemoteConfigRefSpecTest.java

```diff
@@ -0,0 +1,21 @@
+package hudson.plugins.git;
+
+import hudson.util.FormValidation;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class UserRemoteConfigRefSpecTest {
+
+    @Issue("JENKINS-57660")
+    @Test
+    public void testdoCheckRefspec(){
+        String url = "https://github.com/daytonpa/pipeline_test_example.git";
+        String name = "origin";
+        String refSpec = "refs/heads/master";
+        UserRemoteConfig.DescriptorImpl descriptor = new UserRemoteConfig.DescriptorImpl();
+        assertEquals(FormValidation.ok(), descriptor.doCheckRefspec(url, name, refSpec));
+    }
```

Comment:
Thanks for this suggestion Mark, I am adding all of these cases in the test class.

Reply:
@MarkEWaite please review the PR again, I have added more test cases and two localized messages in message.properties.
Also, unmatched wildcards are not accepted by Git while fetching remote repositories, so that is not a success case.